cosmovisor/process.go

```diff
@@ -39,6 +43,15 @@ func LaunchProcess(cfg *Config, args []string, stdout, stderr io.Writer) (bool,
 		return false, fmt.Errorf("launching process %s %s: %w", bin, strings.Join(args, " "), err)
 	}
 
+	sigs := make(chan os.Signal, 1)
+	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
I don't think we should handle SIGINT as a termination signal that causes cosmovisor children to die. I'd rather ignore it actually. Maybe we should handle SIGQUIT instead

Reply:
is it because `Ctrl+C` is already propagated to child process by shell?

Reply:
True that - yes I'm still unsure. You are using `supervisor`, with other process managers/init systems (e.g. systemd) this might not be a problem at all. Plus, what should `cosmovisor` do if the child dies?